src/pybind/mgr/dashboard/services/custom_banner.py

```diff
@@ -0,0 +1,27 @@
+import logging
+from typing import Optional
+
+from mgr_module import _get_localized_key
+
+from .. import mgr
+
+logger = logging.getLogger(__name__)
+
+
+def set_login_banner_mgr(inbuf: str, mgr_id: Optional[str] = None):
+    item_key = 'custom_login_banner'
+    if mgr_id is not None:
+        mgr.set_store(_get_localized_key(mgr_id, item_key), inbuf)
+    else:
+        mgr.set_store(item_key, inbuf)
+
+
+def get_login_banner_mgr():
+    banner_text = mgr.get_store('custom_login_banner')
+    logger.info('Reading custom login banner: %s', banner_text)
+    return banner_text
+
+
+def unsetet_login_banner_mgr():
+    mgr.set_store('custom_login_banner', None)
+    logger.info('Removing custom login banner')
```

Comment:
This is ok, but just for sake of code efficiency: when the services are this kind of trivial (just invoking another service and some logging), you may prescind from the need of an extra service, and call directly the mgr.get/set_store calls, which are in turn services themselves.